src/view/InputView.js

```diff
@@ -0,0 +1,36 @@
+import { Console } from "@woowacourse/mission-utils";
+
+class InputView{
```

Comment:
단순히 함수를 담아두는 역할로 클래스를 사용하시는 이유가 궁금합니다

함수를 보관하기 위한 객체로

```suggestion
const InputView = {
  getInput : function getInput() {
  
  --- 생략---
  }
}
```
와 같이 객체안에 함수를 넣는 것은 어떨까요??

Reply:
모두 class로 쓰다 보니 그렇게 쓰인 것 같습니다.
객체 안에 함수를 넣는 방법 좋은 것 같네요 다음엔 그렇게 써보도록 하겠습니다! 감사합니다!